test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -9,30 +9,37 @@ models:
 traces:
   - name: Fibonacci Waterfall Trace
     model: ref(Waterfall Model)
-    cohort_on: query(year)
-    columns:
+    columns: # Columns default to query
       x_data: category
       y_data: cast( replace(thousands_dollars, ',', '') as float ) * cast(sign as float)
       measure: waterfall
-    props:
+    props: # Props would default to static
       type: waterfall
       base: 0
-      measure: column(measure)
-      x: column(x_data)
-      y: column(y_data)
-      text: query( cast(thousands_dollars as text) )
+      measure:
+        column: measure
+      x:
+        column: x_data
+      y:
+        column: y_data
+      text:
+        query: cast(thousands_dollars as text)
       increasing:
         marker:
           color: "#b97a9b"
+          # Equivalent to:
+          # color:
+          #   static: #b97a9b
       decreasing:
         marker:
           color: "#edbdb5"
-    order_by:
-      - query(row asc)
-    filters:
-      - query( waterfall is not null )
```

Comment:
I think I actually like requiring the query function here. Even though it's more verbose it makes it really easy to glean what's going into the trace query without knowing much about the attributes of the trace